plugins/spark/v3.5/spark/src/main/java/org/apache/polaris/spark/PolarisSparkCatalog.java

```diff
@@ -83,9 +83,24 @@ public Table createTable(
       throws TableAlreadyExistsException, NoSuchNamespaceException {
     try {
       String format = properties.get(PolarisCatalogUtils.TABLE_PROVIDER_KEY);
+
+      String baseLocation;
+      // Extract the base table location from the spark properties.
+      // Spark pass the table base location either with the
+      // TableCatalog.PROP_LOCATION key, or with "path" key if created
+      // with the path option.
+      if (properties.get(TableCatalog.PROP_LOCATION) != null) {
```

Comment:
nit: If location and path are both set, it might be worth a debug log explaining that location is taking precedence